Resources/views/themes/admin/dashboard/block/ez.html.twig

```diff
@@ -0,0 +1,100 @@
+{% trans_default_domain 'dashboard' %}
+
+{# The messages here are on purpose not translated as some (+ additional) are inteded to be retrived from updates.ez.no in the future #}
+
+{% set badge = "" %}
+{% set status %}
+    {% if not ez.release %}
+        <div class="alert alert-warning" role="alert">
+            🤷 Could not find your <code>composer.lock</code> file, it's needed in order to determine information about
+             your eZ install, and recommended to be kept on project development to make sure same package versions are used across all environments.
+        </div>
+    {% elseif not ez.debug and ez.stability != 'stable'  %}
+        <div class="alert alert-warning" role="alert">
+            💁 Your setup is running with unstable packages.
+            {% if ez.isEnterpise %}
+                This is not supported unless told otherwise, if you need assistance <a href="https://support.ez.no">get in touch</a>.
```

Comment:
```suggestion
                This is not supported unless communicated otherwise. If you need assistance, <a href="https://support.ez.no">get in touch with us</a>.
```